Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.12.8-alpine
+LABEL maintainer="sberdianskyi@mate.com"
+ENV PYTHONBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
```

Comment:
The `COPY` command should specify the source and destination more clearly. For example, use `COPY requirements.txt /app/` to ensure the file is copied to the correct directory.